lib/ftp.c

```diff
@@ -3292,9 +3292,18 @@ static CURLcode ftp_done(struct connectdata *conn, CURLcode status,
 
     if(!ftpc->dont_check) {
       /* 226 Transfer complete, 250 Requested file action okay, completed. */
-      if((ftpcode != 226) && (ftpcode != 250)) {
```

Comment:
I think a switch statement makes this more complex. What's against an or 2 if clauses?

Reply:
To me, a switch() is *simpler* than a three-clause if() chain.